proposed/http-middleware/middleware-meta.md

```diff
@@ -123,7 +130,7 @@ Middleware taking this approach generally has the following commonalities:
   the request for processing.
 * The middleware is passed 2 arguments during invocation:
   1. An object that represents an HTTP request.
-  2. A delegate that receives the request to dispatch next middleware in the pipeline.
+  2. A handler that receives the request to dispatch next middleware in the pipeline.
```

Comment:
This implies architecture: that the handler passed necessarily composes middleware and/or has a queue or stack of middleware. The handler _could_ simply return a response.

Perhaps another way to phrase it: "A handler that may be called in order to produce a response for the middleware."